### PR TITLE
Feature/123 footer styling

### PR DESCRIPTION
**Pull request by RandomTannenbaum**

### CODE REVIEW

File: src/main/resources/templates/fragments/footer.html [hunk 1]
```diff
@@ -1,11 +1,10 @@
-<footer class="w-100 position-absolute z-1 bottom-0 bg-white">
-    <hr class="mt-0">
-    <div class="p-3 align-items-center justify-content-between d-flex">
+<footer class="w-100 position-absolute z-1 bottom-0 bg-white d-flex justify-content-center">
+    <div class="align-items-center justify-content-between d-flex border-top border-black w-100 h-100 p-4 ms-3 me-3">
```
Comment (Miguel7373) [inline src/main/resources/templates/fragments/footer.html:2]: ms und me mit mx lösen

File: src/test/e2e/cypress/e2e/footer.cy.js [hunk 2]
```diff
@@ -18,4 +18,10 @@ it('should redirect to impressum', () => {
     cy.get("footer").find("a").eq(1)
         .should('have.attr', 'href')
         .and('include', 'https://www.unilu.ch/impressum/');
-});
\ No newline at end of file
+});
+
+it('should redirect to github-page', () => {
```
Comment (RandomTannenbaum) [inline src/test/e2e/cypress/e2e/footer.cy.js:23]: The test name is unfitting as it only checks for the href attribute and not if the redirect actually happens.

File: src/main/resources/templates/fragments/footer.html [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+    <div class="align-items-center justify-content-between d-flex border-top border-black w-100 h-100 p-4 mx-3">
         <a data-testid="swissuniversities-link" href="https://www.swissuniversities.ch/" target="_blank">
             <img data-testid="swissuniversities-logo" alt="Swissuniversities logo" height="22px" th:src="@{images/swissuniversities.png}">
         </a>
-        <div class="d-flex justify-content-end gap-4">
-            <a data-testid="impressum-link" class="link-primary" href="https://www.unilu.ch/impressum/" target="_blank">Impressum & Datenschutz</a>
-            <span th:text="${'Version ' + @environment.getProperty('spring.application.version')}"></span>
+        <div class="d-flex align-items-center gap-4">
+            <a class="link-primary" href="https://www.unilu.ch/impressum/" target="_blank" data-testid="impressum-link">Impressum & Datenschutz</a>
+            <p class="m-0" th:text="${'Version ' + @environment.getProperty('spring.application.version')}" data-testid="version-link"></p>
```
Comment (MasterEvarior) [inline src/main/resources/templates/fragments/footer.html:8]:
```suggestion
            <p class="m-0" th:text="${'Version ' + @environment.getProperty('spring.application.version')}" data-testid="version"></p>
```
Maybe we should rename this to just `version` as it is no longer a link.

**Review by MasterEvarior — APPROVED** [review]
Just one nitpick about the naming, everything else looks fine.

**Review by MasterEvarior — APPROVED** [review]

**Pull request merged**